src/views/HomeView.vue

```diff
@@ -4,17 +4,36 @@ import Button from '@/components/SButton.vue'
 import EventCard from '@/components/SEventCard.vue'
 import Header from '@/components/SHeader.vue'
 import HeaderItem from '@/components/SHeaderItem.vue'
+import SidebarItem from '@/components/SSidebarItem.vue'
+import HomeIcon from '@/components/icons/HomeIcon.vue'
+import SearchIcon from '@/components/icons/SearchIcon.vue'
+import LoginIcon from '@/components/icons/LoginIcon.vue'
 import MlbIcon from '@/components/icons/MlbIcon.vue'
 import NbaIcon from '@/components/icons/NbaIcon.vue'
 import NflIcon from '@/components/icons/NflIcon.vue'
 import NhlIcon from '@/components/icons/NhlIcon.vue'
 
+
 const axios = inject('axios')
 const allSports = ref([])
 const events = ref([])
 const bets = ref([])
 const teams = ref([])
 const sports = ref([])
+const links = ref([
+  {
+    text: 'home',
+    icon: null
+  },
+  {
+    text: 'search',
+    icon: null
+    },
+    {
+      text: 'login',
+      icon: null
+      }
```

Comment:
Formatting looks a little weird here.

Reply:
Since you're building out your links with icons below, then this isn't necessary.

Instead you can just make links be an array of the names of the links, then build the link with the icons below.